frontend/src/pages/item/_itemId/index.vue

```diff
@@ -246,13 +247,24 @@ import {
 import { getUserLibraryApi } from '@jellyfin/sdk/lib/utils/api/user-library-api';
 import { getBlurhash } from '@/utils/images';
 import { getItemDetailsLink, getMediaStreams } from '@/utils/items';
-import { getItemizedSelect } from '@/utils/forms';
+import { getItemizedSelect, VSelectItem } from '@/utils/forms';
```

Comment:
Fix the lint issue from here, please

Reply:
Good catch, I must have been tired I forgot many of them ^^'